### PR TITLE
Changing keyboard type in ThresholdsSettingsView

### PR DESCRIPTION
**Pull request by SzymonGorowski**
https://trello.com/c/4qBI79YP/650-changing-keyboard-type-in-thresholds-settings-view

Two ways of handling the keyboard because the toolbar for the keyboard is available only for iOS 15.0 and higher. So for iOS > 15 users will tap "Done" to dismiss the keyboard as per the attached screenshot. Whereas for iOS versions lower than 15 the user will tap anywhere to dismiss the keyboard.

screenshot

### CODE REVIEW

File: AirCasting/SessionViews/ThresholdSettings/ThresholdsSettingsView.swift [hunk 1]
```diff
@@ -22,51 +22,114 @@ struct ThresholdsSettingsView: View {
...
+                }
+                .padding()
+                
+                Section {
+                    veryHighTextfield
+                    highTextfield
+                    mediumTextfield
+                    lowTextfield
+                    veryLowTextfield
+                }
```
Comment (jankrzempek) [inline AirCasting/SessionViews/ThresholdSettings/ThresholdsSettingsView.swift:43]: All this chunk of code can be even shorter I think

```
var body: some View {
        if #available(iOS 15.0, *) {
            mainBody
            .toolbar {
                ToolbarItemGroup(placement: .keyboard) {
                    Spacer()
                    Button("Done") { hideKeyboard() }
                }
            }
        } else {
            mainBody
            .onTapGesture { hideKeyboard() }
        }
    }
```

where mainBody is:

```
 var mainBody: some View {
        Form {
            VStack(alignment: .leading, spacing: 16) {
                Text(Strings.SessionCart.heatmapSettingsTitle)
                    .foregroundColor(.darkBlue)
                    .font(Fonts.heavyTitle1)
                Text(Strings.SessionCart.heatmapSettingsdescription)
                    .foregroundColor(.aircastingGray)
                    .font(Fonts.regularHeading2)
            }
            .padding()
            
            Section {
                veryHighTextfield
                highTextfield
                mediumTextfield
                lowTextfield
                veryLowTextfield
            }
            .keyboardType(.numberPad)
            
            VStack {
                Button(action: {
                    thresholdValues = thresholdSettingsViewModel.updateToNewThresholds()
                    presenta....
```

**Review by jankrzempek — CHANGES_REQUESTED** [review]
Work great, two things only 😇

File: AirCasting/SessionViews/ThresholdSettings/ThresholdsSettingsView.swift (hunk 1 above)
Comment (SzymonGorowski) [reply AirCasting/SessionViews/ThresholdSettings/ThresholdsSettingsView.swift:43]: Nice catch, will be corrected.

**Review by anna1901 — APPROVED** [review]
Awesome, great changes! Good that you added dissmissing the keyboard

**Pull request merged**